website_country_localized_pages/controllers/main.py

```diff
@@ -0,0 +1,41 @@
+# -*- coding: utf-8 -*-
+#
+#    © 2015 Agile Business Group (<http://www.agilebg.com>)
+#    License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+#    See __openerp__.py file
+#
+
+from openerp.addons.website.controllers.main import Website
+from openerp.addons.web import http
+from openerp.http import request
+import logging
+_logger = logging.getLogger(__name__)
+try:
+    from ipwhois import IPWhois
+    from ipwhois import IPDefinedError
+except ImportError:
+    _logger.debug("Can not 'import ipwhois'.")
```

Comment:
[Import ordering and separation](https://github.com/OCA/maintainer-tools/blob/master/CONTRIBUTING.md#imports)